docs/series-snapshot-redesign.md

```
@@ -0,0 +1,263 @@
+- `SeriesCompetitor`
+- `SeasonSeries`
+- `SeasonSeriesCompetitor`
+
+Columns on `BaseballCompetition`:
+- `CurrentSeriesId` (FK → `Series.Id`)
+- `SeasonSeriesId` (FK → `SeasonSeries.Id`)
+
+### Adds (all on `BaseballCompetition`)
+
+Grouping key:
+| Column | Type | Notes |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)